.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,34 @@
+name: Regenerate WPT certificates
+on:
+  schedule:
+    # Our certificates are good for a few years, but we regenerate them every 6
+    # months to be sure.
+    - cron: '0 0 1 */6 *'
+jobs:
+  update:
+    runs-on: ubuntu-18.04
```

Comment:
Nit: maybe try 20.04 as it's the new LTS?

Reply:
Based on https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners, should probably use `ubuntu-latest` and it'll update to 20.04 when it is out of preview. I'll make the switch.